File: 1051.c
#include <stdio.h>

int main()
{
	double input, out;
	
	scanf("%lf", &input);
	
	if(input <= 2000)
	{
		printf("Isento\n");
	} else if(input <= 3000)
	{
		out = (input - 2000) * 0.08;
		printf("R$ %.2lf\n", out);
	} else if(input <= 4500)
	{
		out = (input - 3000) * 0.18 + 80;
		printf("R$ %.2lf\n", out);
	}	else 
	{
		out = (input - 4500) * 0.28 + 80 + 270;
		printf("R$ %.2lf\n", out); 
	}
	
	return 0;
}

File: python/1051.py
d = float(input())

if d > 0 and d <= 2000:
    print("Isento")
elif d > 2000 and d <= 3000:
    r = d - 2000
    s = r * 0.08
    print("R$ %.2f" % s)
elif d > 3000 and d < 4500:
    r = d - 3000
    s = (r * 0.18) + (1000 * 0.08)
    print("R$ %.2f" % s)
else:
    r = d - 4500
    s = (r * 0.28) + (1500 * 0.18) + (1000 * 0.08)
    print("R$ %.2f" % s)
